ratis-logservice/src/test/java/org/apache/ratis/logservice/LogServiceReadWriteBase.java

```diff
@@ -64,6 +69,46 @@
   static final int NUM_PEERS = 3;
   CLUSTER cluster;
 
+  class MetricLogStream extends LogStreamImpl{
```

Comment:
Actually required for testing only, this is just counting the calls made through log stream, so that we can compare the count with the metrics collected at the server.

Reply:
Ok, cool!